ada/views.py

```diff
@@ -0,0 +1,28 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import HttpResponseRedirect, render
+
+from .models import RedirectUrl, RedirectUrlsEntry
+
+
+def clickActivity(request: HttpRequest, pk: int) -> HttpResponse:
+    try:
+        redirect_url = RedirectUrl.objects.get(id=pk)
+        store_url_entry = RedirectUrlsEntry(url=redirect_url)
+        store_url_entry.save()
+        destination_url = (
+            f"{redirect_url.symptom_check_url}"
+            f'/?whatsappid={request.GET.get("whatsappid")}'
+        )
+        return HttpResponseRedirect(destination_url)
+    except Exception as e:
```

Comment:
We don't want these generic catching of exceptions, rather catch ones we can handle properly, and let the others go into sentry so that we can know that they exist and fix the bug that's causing them